src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -224,19 +218,19 @@ static void setup() {
   @Test
   @DisplayName("Test Convert to CDF works for CDF")
   void convertToCdfFromCdf() {
-    runConvertToCdfTest("convert_to_cdf_from_cdf");
+    runConvertToCdfAndCsvTest("convert_to_cdf_from_cdf");
   }
 
   @Test
   @DisplayName("Test Convert to CDF works for Dominion")
   void convertToCdfFromDominion() {
-    runConvertToCdfTest("convert_to_cdf_from_dominion");
+    runConvertToCdfAndCsvTest("convert_to_cdf_from_dominion");
   }
 
   @Test
   @DisplayName("Test Convert to CDF works for ES&S")
   void convertToCdfFromEss() {
-    runConvertToCdfTest("convert_to_cdf_from_ess");
+    runConvertToCdfAndCsvTest("convert_to_cdf_from_ess");
```

Comment:
It's trivial to add 100% test coverage here, but since this file has the full CSVs, it causes significant repo bloat -- CVRs essentially double in size, one for input one for output.

In tandem with the tests of outputs, I feel sufficiently confident that this should cover all our bases...let me know if you prefer some other direction here!

Reply:
I trust your judgment here. :)